src/components/Header/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+
+const Header = (props) => {
+  const {countCartItems} = props;
+  const navStyle = { color: 'white' };
+
+  return(
+    <nav>
+      <Link style={navStyle} to='/'>
+        <h1>Home</h1>
+      </Link>
+      <ul className="nav-links">
+        <Link style={navStyle} to='/cart'>
+          <li> Cart{' '}
+            {countCartItems ? (
+              <button className="badge badge-success badge-pill">{countCartItems}</button>
```

Comment:
same about the &&, and also you can use &nbsp; to add the white space instead of the `{' '}`